docs/release_notes/0.0.105.md

```diff
@@ -2,6 +2,8 @@
 
 ## Features
 
+okctl#1021 ✅ Add encryption to node volumes
```

Comment:
As this is the okctl repo, isn't it enough with just `#1021 ✅ Add encryption to node volumes` ?

Reply:
Thought was to seperate issues between the multiple repos we have in the project board. #1021 is probably enough yes